realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -281,6 +283,40 @@ public void refresh() {
     }
 
     /**
+     * Blocks the current thread until transactions are committed by other threads to this Realm or
+     * {@link #stopWaitForChange()} is called from other threads.
+     *
+     * @return {@code true} if transactions are committed to this Realm instance, {@code false} if
```

Comment:
What about:

`{@code true If the Realm was updated to the latest version, {@code false} if it was cancelled by calling {@link #stopWaitForChange}.`